src/rpc/RPCClient.ts

```diff
@@ -88,7 +88,7 @@ class RPCClient<M extends ClientManifest> {
     {},
     {
       get: (_, method) => {
-        if (typeof method === 'symbol') throw never();
+        if (typeof method === 'symbol') return;
```

Comment:
It would end up throwing if you ever `console.log()` the proxy or class. So I changed it to return nothing.

Reply:
Should we even allow such methods?

Reply:
I'm not sure what you mean.

I think just returning undefined is fine here, since it's just would be undefined. In fact, we don't even need this line it would be covered by the default case.

Reply:
Nevermind, it's needed to enforce types for the line `this.callerTypes[method]`.

Reply:
So what happens if the method is a symbol? Is this just a formality? Is there some runtime behaviour here?

Reply:
It would just be undefined rather than an error. which is normal here unless there are some implicit parameters for objects.